src/sagemaker/script_uris.py

```diff
@@ -27,6 +28,8 @@ def retrieve(
     model_id=None,
     model_version=None,
     script_scope=None,
+    tolerate_vulnerable_model: Optional[bool] = None,
+    tolerate_deprecated_model: Optional[bool] = None,
```

Comment:
same comment.

Reply:
I'd rather keep the pattern of setting these params to None.